build/build.proj

```diff
@@ -36,6 +37,14 @@
              Targets="Build"
              Properties="$(CommonMSBuildGlobalProperties)"
              />
+
```

Comment:
This target is not under RestorePackages. The github diff gave you the illusion :)

But I will move it to a separate target